email.js

```diff
@@ -18,14 +38,10 @@ function parseBody(body) {
     const res = {
         subject,
     };
-    for (const part of parsed.childNodes) {
-        const mtype = part.headers['content-type'][0].value;
-        if (mtype === 'text/plain') {
-            res.text = (new TextDecoder(part.charset)).decode(part.content);
-        } else if (mtype === 'text/html') {
-            res.html = (new TextDecoder(part.charset)).decode(part.content);
-        }
-    }
+
+    const parse_result = parseDeep(parsed);
```

Comment:
A more JavaScript-y way of writing this would be
```
const {text, html} = parseDeep(parsed);
```

Reply:
```
    const { text, html } = parseDeep(parsed);
    res.text = text;
    res.html = html;
```
?

Reply:
Yes (with different spacing between braces, but we need to add tooling support to get that by default in some editors).